### ISSUE TITLE
Streched-Exponential and Power law+cut off distributions

### ISSUE THREAD

**Issue by dbmehta** [csgillespie/poweRlaw#63]
Hi,
Is there any chance you would include other distributions such as stretched-exponential and power law+cut off some time soon? Adding at least these two will make the package a complete user-friendly and one-stop-shop for the power-law comparison for the end-users like myself. Such a package is highly desirable and timely.
Thanks.

**Comment by csqsiew** [comment]
agreed, hope that these distributions will be added to the package soon.

**Comment by tomleung1996** [comment]
agree

**Comment by csgillespie** [comment]
Sorry, I no longer have time to actively add new functionality to this package.
